elements/lisk-transaction-pool/test/unit/transaction_pool.spec.ts

```diff
@@ -343,6 +343,133 @@ describe('TransactionPool class', () => {
 			expect(status).toEqual(Status.FAIL);
 		});
 
+		it('should evict the trx from the pool when txPool is full when all the trxs are processable', async() => {
```

Comment:
```suggestion
		it('should evict lowest feePriotity among highest nonce trx from the pool when txPool is full when all the trxs are processable', async() => {
```

Reply:
Good suggestion maybe also change `when all the trxs are processable` to `and all the trxs are processable`